docs/DeveloperGuide.md

```diff
@@ -255,44 +255,104 @@ _{Explain here how the data archiving feature will be implemented}_
+| Priority | As a …​                                              | I want to …​                                                                       | So that I can…​                                                                                               |
+|----------|------------------------------------------------------|------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------|
+| `* * *`  | new user                                             | see usage instructions                                                             | refer to instructions when I forget how to use the App                                                        |
+| `* * *`  | frequent user                                        | update new data immediately                                                        | I can exit and enter the product freely                                                                       |
+| `* * *`  | frequent user                                        | enter and exit the product in a short amount of time                               | use the product on the fly                                                                                    |
+| `* * *`  | marketer                                             | add a new customer contact                                                         | keep track of all customers and enlarge the customer base over time                                           |
+| `* * *`  | marketer                                             | delete a customer contact                                                          | remove entries that I no longer need                                                                          |
+| `* * *`  | marketer                                             | find a customer contact by name                                                    | locate details of persons without having to go through the entire list                                        |
+| `*`      | marketer                                             | hide private customer contact details                                              | minimize chance of someone else seeing them by accident                                                       |
+| `* * *`  | pre-event marketer                                   | view all the upcoming marketing events in the system sorted in chronological order | have a high level overview of the frequency of the marketing events and be mentally prepared for each of them |
+| `* * *`  | pre-event marketer who handles many marketing events | add marketing events                                                               | keep the marketing system up to date with changes that happen to the marketing plans                          |
```

Comment:
Small change to be consistent with the previous story's centre column which mentions "the system".

```suggestion
| `* * *`  | pre-event marketer who handles many marketing events | add marketing events                                                               | keep the system up to date with changes that happen to the marketing plans                          |
```